book/03-git-branching/sections/nutshell.asc

```diff
@@ -124,6 +124,16 @@ Let's switch back to the `master` branch:
 $ git checkout master
 ----
 
+[NOTE]
+.`git log` doesn't show _all_ the branches _all_ the time
```

Comment:
Let's remove the block header here. We don't use them anywhere else, and I'm honestly not sure how it will render in every format.

Reply:
Hmm... I copied the format of the note from [a previous note](https://github.com/progit/progit2/blame/master/book/03-git-branching/sections/nutshell.asc#L135) in this same section, so we might have to do this a few more places.

I'm not quite sure what you mean. Should I just get rid of "[NOTE]"?

Reply:
Whoops, you're absolutely right, sorry about that! It's totally fine the way it is.

Reply:
Whoops, you're absolutely right, sorry about that! It's totally fine the way it is.